src/mono/mono/mini/cpu-amd64.md

```diff
@@ -815,7 +815,7 @@ expand_i8: dest:x src1:i len:11
 expand_r4: dest:x src1:f len:16
 expand_r8: dest:x src1:f len:13
 
-roundpd: dest:x src1:x len:10
+roundp: dest:x src1:x len:10
```

Comment:
Just a minor clean up - I made `roundps`, `roundpd` just `roundp` with type in `inst_c1`
It helped me to reduce a lot of copy-paste in Sse41 - it uses `roundps`, `roundpd`, `roundss`, `roundsd` A LOT.